projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/index.tsx

```diff
@@ -1,20 +1,47 @@
 import { __ } from '@wordpress/i18n';
-import { PRODUCT_STATUSES } from '../../../constants';
+import { useCallback, type FC } from 'react';
+import useProduct from '../../../data/products/use-product';
+import useAnalytics from '../../../hooks/use-analytics';
 import ProductCard from '../../connected-product-card';
 import ProtectValueSection from './protect-value-section';
-import type { FC } from 'react';
 
 const ProtectCard: FC< { admin: boolean } > = ( { admin } ) => {
-	// Override the primary action button to read "Protect your site" instead
-	// of the default text, "Learn more".
-	const primaryActionOverride = {
-		[ PRODUCT_STATUSES.ABSENT ]: {
-			label: __( 'Protect your site', 'jetpack-my-jetpack' ),
-		},
+	const { recordEvent } = useAnalytics();
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { hasPaidPlanForProduct: hasProtectPaidPlan } = detail;
+
+	/**
+	 * Called when secondary "View" button is clicked.
+	 */
+	const onViewButtonClick = useCallback( () => {
+		recordEvent( 'jetpack_myjetpack_product_card_manage_click', {
+			product: slug,
+		} );
+	}, [ recordEvent ] );
+
+	const shouldShowSecondaryButton = useCallback(
+		() => ! hasProtectPaidPlan,
+		[ hasProtectPaidPlan ]
+	);
+
+	const viewButton = {
+		href: 'admin.php?page=jetpack-protect',
+		label: __( 'View', 'jetpack-my-jetpack' ),
+		onClick: onViewButtonClick,
+		shouldShowButton: shouldShowSecondaryButton,
 	};
 
+	const noDescription = useCallback( () => null, [] );
```

Comment:
Maybe we can leave a comment for the future reference that if we gonna remove description to more cards, we should consider extending `<ProductCard />` functionality to support that.

Clever workaround here though! 🙌